packages/react/src/avatar/utils.ts

```diff
@@ -1,7 +1,5 @@
 export function getInitialsFromName(name: string) {
-	const matches = Array.from(
-		name.matchAll(new RegExp(/(\p{L}{1})\p{L}+/, 'gu'))
-	);
+	const matches = Array.from(name.matchAll(new RegExp(/(\p{L})\p{L}+/, 'gu')));
```

Comment:
I don't think this regular expression is correct to start with. It won't pick up single letters so `O'` and the like are skipped. `O'Shea`, `O'Malley`, `O'Keefe`, `D'arcy` etc. French names do this a bit, I think.

Reply:
There's other things, too, like `Sr`. Anyway, probably for another ticket.

Reply:
Good call out. Perhaps create a ticket?